cloud/aws/policies.tf

```diff
@@ -122,6 +115,32 @@ resource "aws_iam_policy" "sfx_metric_streams_policy" {
 EOF
 }
 
+resource "aws_iam_policy" "sfx_logs_policy" {
+  count       = var.create_logs_iam ? 1 : 0
+  name        = "SignalFxIntegrationLogsPolicy${var.suffix == "" ? "" : "-${title(var.suffix)}"}"
+  description = "SignalFx AWS Policy for ingesting Cloudwatch Logs"
+  policy      = <<EOF
+{
+"Version": "2012-10-17",
+"Statement": [
+    {
+        "Action": [
+            "logs:DeleteSubscriptionFilter",
+            "logs:DescribeLogGroups",
+            "logs:DescribeSubscriptionFilters",
+            "logs:PutSubscriptionFilter",
+            "s3:GetBucketLogging",
+            "s3:GetBucketNotification",
+            "s3:PutBucketNotification"
+        ],
+        "Effect": "Allow",
+        "Resource": "*"
```

Comment:
Can't we target specific Cloudwatch logs and S3 resources?

Reply:
I don't think so given that each service can store logs in different buckets (e.g. for ELBs) and splunk does not target anything from their official cloudformation template